build-with-pip.file

```diff
@@ -24,7 +24,7 @@
 %endif
 
 %if "%{?pip_name:set}" != "set"
-%define pip_name %(echo %n | cut -f2-10 -d-)
+%define pip_name %(echo %n | cut -f2-10 -d- | tr '-' '_')
```

Comment:
@iarspider , why this change was needed? Ideally we should have the correct package name in requirement.txt file i.e. `%n` should already be the correct.

Note that this change means all the `py3-*` packages will be rebuilt. If this change is required then we should also see if we can update pip and setuptools.

Reply:
@smuzaffar because this is how tarballs are named: the packages' names have `-`, but tarballs have `_`.